### PR TITLE
#1121 Added Jobs, Speed to People

### PR DESCRIPTION
**Pull request by carlosmiranda**
#1121: Added Jobs and Speed to `People.java`.

* Updated Datum to support new XML schema that includes `speed` element.
* Added `jobs` and `speed` to `People.java`
* Added tests
* Added puzzle for final implementation of speed.**

** Note: There is already an existing puzzle for adding jobs/agenda to `people.xml`. See #841.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1251](https://github.com/zerocracy/farm/issues/1251) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1251](https://github.com/zerocracy/farm/issues/1251) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1770151917); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 1]
```diff
@@ -425,6 +426,34 @@ public void setsDetails() throws Exception {
...
+        people.jobs(uid, jobs);
+        MatcherAssert.assertThat(
+            people.jobs(uid),
+            Matchers.is(jobs)
+        );
+    }
+
+    @Test
+    public void setsSpeed() throws Exception {
+        final FkFarm farm = new FkFarm(new FkProject());
```
Comment (krzyk) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @carlosmiranda farm should be inlined

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 2]
```diff
@@ -425,6 +426,34 @@ public void setsDetails() throws Exception {
         );
     }
 
+    @Test
+    public void setsJobs() throws Exception {
+        final FkFarm farm = new FkFarm(new FkProject());
```
Comment (krzyk) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @carlosmiranda farm should be inlined

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 3]
```diff
@@ -425,6 +426,34 @@ public void setsDetails() throws Exception {
         );
     }
 
+    @Test
```
Comment (krzyk) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @carlosmiranda I think we should also have a test for user that doesn't exist for both speed and job

**Comment by krzyk** [comment]
@carlosmiranda and also the build fails because of qulice violations

**Comment by carlosmiranda** [comment]
@krzyk updated, please check. I also had to update the Checkstyle suppressions in `PeopleTest`. Note that `ClassDataAbstractionCouplingCheck` seems to only need suppression to the class declaration which is why it's now "3 lines".

**Comment by krzyk** [comment]
@carlosmiranda thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1251](https://github.com/zerocracy/farm/issues/1251) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
> @0crat quality good ([here](https://github.com/zerocracy/farm/issues/1251#issuecomment-400794855))

@ypshenychka There is no quality review for [#1251](https://github.com/zerocracy/farm/issues/1251), no performer

**Comment by 0crat** [comment]
Order was finished, quality is "good": +23 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by krzyk** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/1251#issuecomment-400819962))

@krzyk This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1251](https://github.com/zerocracy/farm/issues/1251) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%231251%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Pull request merged**